src/components/ExtractorResponseEditor/ExtractorResponseEditor.css

```diff
@@ -28,10 +28,15 @@
 .entity-labeler__custom-editor--read-only {
     background-color: rgba(0,0,0,0.025);
 }
+
 .entity-labeler__custom-editor--error {
     border-color: red;
```

Comment:
Hmm, not related to PR but I noticed this when you were demoing.  This should be the `var(--color-error)` from Office fabric to match our other errors.